pkg/operation/botanist/controlplane/clusterautoscaler/monitoring.go

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package clusterautoscaler
+
+import (
+	"bytes"
+	"strings"
+	"text/template"
+
+	v1beta1constants "github.com/gardener/gardener/pkg/apis/core/v1beta1/constants"
+
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+)
+
+const (
+	monitoringPrometheusJobName = "cluster-autoscaler"
+
+	monitoringMetricProcessMaxFds  = "process_max_fds"
+	monitoringMetricProcessOpenFds = "process_open_fds"
```

Comment:
I just spotted that these constants are duplicated in the kube-scheduler package ( and I am about to use them aswell for the kcm).
I think it would make sense to put them into a common package.

Reply:
@wyb1 does this make sense? I would guess these metrics are component-specific and they are just "by accident" the same?

Reply:
These metrics are exposed by most exporters so we can add them to a common package as long as we can decide to add them for the component (or not).

Reply:
Hm, ok, thanks, I see. I would still prefer to keep them here for now as I wouldn't know where to put these (which "common" package) - for now it should be more clear as it is.